ImageInNavigationBarDemo/DemoTableViewController.swift

```diff
@@ -7,3 +7,157 @@
 //
 
 import UIKit
+
+class DemoTableViewController: UITableViewController {
+
+    let imageView = UIImageView(image: UIImage(named: "image_name"))
+
+    /// WARNING: Change these constants according to your project's design
+    private struct Const {
+        /// Image height/width for Large NavBar state
+        static let ImageSizeForLargeState: CGFloat = 40
+        /// Spacing from right anchor of safe area to right anchor of Image
+        static let ImageRightSpace: CGFloat = 16
+        /// Spacing from bottom anchor of NavBar to bottom anchor of Image for Large NavBar state
+        static let ImageBottomSpaceForLargeState: CGFloat = 12
+        /// Spacing from bottom anchor of NavBar to bottom anchor of Image for Small NavBar state
+        static let ImageBottomSpaceForSmallState: CGFloat = 6
+        /// Factor used to multiply your ImageSizeForLargeState to get the Image Size for Small NavBar State. For example: Image size for Large NavBar is 40, Image size for Small NavBar is 32, then value of SmallImageFactor = 32/40 = 0.8
+        static let SmallImageFactor: CGFloat = 0.8
```

Comment:
What you think about using the size of the small image instead of the factor as the constant? It can be more convenient for the readers and users

Reply:
yeah makes sense. and then calculate the factor separately right?